src/windows.rs

```diff
@@ -64,6 +64,43 @@ pub fn apply_mica(hwnd: HWND) {
   }
 }
 
+pub fn clear_mica(hwnd: HWND) {
+  if is_win11_dwmsbt() {
+    unsafe {
+      DwmSetWindowAttribute(
+        hwnd,
+        DWMWA_SYSTEMBACKDROP_TYPE,
+        &(DWM_SYSTEMBACKDROP_TYPE::DWMSBT_DISABLE as i32) as *const _ as _,
+        4,
+      );
+    }
+  } else if is_win11() {
+    unsafe {
+      DwmSetWindowAttribute(hwnd, DWMWA_MICA_EFFECT, &1 as *const _ as _, 4);
+    }
+  } else {
+    eprintln!("\"clear_mica\" is only available on Windows 11");
+  }
+}
+
+pub fn clear_effects(hwnd: HWND) {
```

Comment:
split into `clear_blur` and `clear_acylic` and check for Windows versions, same as `apply_*` variants

Reply:
Do I need different version checking? The functions are using the same checks as `apply_*`s, unless I'm mistaken

Reply:
`clear_blur` and `clear_mica` is fine now. I leave you a comment on `clear_acrylic`.